lib/rspec/hanami.rb

```diff
@@ -1,8 +1,8 @@
 require 'rspec/hanami/version'
 require 'rspec/hanami/request_helpers'
+require 'rspec/hanami/matchers'
 
-module RSpec
-  module Hanami
-    autoload :Matchers, 'rspec/hanami/matchers'
-  end
+RSpec.configure do |config|
+  config.include RSpec::Hanami::Matchers, type: :request
+  config.include RSpec::Hanami::RequestHelpers, type: :request
```

Comment:
not use that it's correct lines.
We use RequestHelpers helper in `feature` types spec. And `Matchers` in `action`, `view`, `entity` and other types.

Could you fix it before merge please?

Reply:
sure

Reply:
A little bit confused by 

> And Matchers in action, view, entity and other types.

Here is my suggestion:
```suggestion
  config.include RSpec::Hanami::RequestHelpers, type: :request
  config.include RSpec::Hanami::RequestHelpers, type: :feature

  config.include RSpec::Hanami::Matchers, type: :request
  config.include RSpec::Hanami::Matchers, type: :action
  config.include RSpec::Hanami::Matchers, type: :view
  config.include RSpec::Hanami::Matchers, type: :entity
```

Reply:
Also, I found another one unexpected behavior: when I'm require matchers and run all tests some tests expects from subject to be an array (rack-response)

```ruby
subject { interactor.() }
it { expect(subject).to be_success }
```

and error message

> NoMethodError: undefined method `first' for #<Hanami::Interactor>

This error is referenced to `Hanami::Matchers::BeStatus`